### PR TITLE
Fix blending and module more menu buttons

### LINKED ISSUE #6826: better blending mode widget layout
**Is your feature request related to a problem? Please describe.**
better blending mode widget layout

**Describe the solution you'd like**
the current button (last on the right) is smaller than others, I'm sure we can do better, see:

image

### PR DESCRIPTION
**Pull request by Nilvus**
Fix #6826

@elstoc: your feedback is welcome, as yours @TurboGit

### CODE REVIEW

**Comment by elstoc** [comment]
fine for me

File: data/themes/darktable.css [hunk 1]
```diff
@@ -979,10 +979,16 @@ notebook tabs,
...
 	font-size: 0.8em;
-	padding: 0.1em;
+	padding: 0.25em;
 	margin: 0 0.5em;
 }
 
+#blending-tabs #dt-button,
+#modules-tabs #dt-button
+{
+	min-width: 1.25em;
```
Comment (TurboGit) [inline data/themes/darktable.css:989]: It is still visually smaller, I tried 1.4 and seems better to me.

**Comment by TurboGit** [comment]
BTW, we have the same issue with the module groups buttons. The last one on the right is way to close to the border:

image

File: data/themes/darktable.css (hunk 1 above)
Comment (Nilvus) [reply data/themes/darktable.css:988]: I disagree. This button needs to be a little smaller than main buttons.

**Comment by Nilvus** [comment]
@TurboGit: check last update. All should be better.

**Review by TurboGit — APPROVED** [review]
Yes, better for me now for both buttons. Thanks!

**Pull request merged**